app/services/encryption/kms_logger.rb

```diff
@@ -11,7 +11,11 @@ def self.log(action, context = nil)
     end
 
     def self.logger
-      @logger ||= Logger.new('log/kms.log')
+      if Figaro.env.log_to_stdout?
```

Comment:
I'd suggest we do this through the FeatureManagement class and add a little extra logic so that in the test environment, we still log to files --- the STDOUT logging spams during the test suite and it's really hard to see what's going on in the CircleCI logs, something like

```ruby
def log_to_stdout?
  !Rails.env.test? && Figaro.env.log_to_stdout?
end
```